run.sh

```diff
@@ -2,6 +2,47 @@
 
 set -o errexit
 
+usage() {
+    echo "deploy               -- Deploy airship"
+    echo "update_openstack     -- Update openstack deployment"
```

Comment:
openstack->OpenStack.

Reply:
it wont be consistent rest of options? I think we either make all small case or all Mixed cases ?

Reply:
I think James is referring to the documentation side. ie "Update OpenStack deployment".

Reply:
> I think James is referring to the documentation side. ie "Update OpenStack deployment".

Yes its already fixed.